pkg/ccl/importccl/read_import_mysql.go

```diff
@@ -345,9 +362,7 @@ func mysqlTableToCockroach(
 			return nil, nil, err
 		}
 		if raw.Type.Autoincrement {
-			if seqName == "" {
```

Comment:
Is it possible for autoincrement to appear twice in a table?

Reply:
no